src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -58,51 +53,71 @@ protected OutputSettings() {
      * for video converter.
      */
     protected void setResolutionSelect() {
-        resolutionSelect.addItem("Original resolution");
-        resolutionSelect.addItem("HD720p\t1280x720");
-        resolutionSelect.addItem("1920p\t108x1920");
-        resolutionSelect.addItem("480p\t854x480");
-        resolutionSelect.addItem("240p\t426x240");
-        resolutionSelect.addItem("DVD\t720x567");
-        resolutionSelect.addItem("Television\t640x480");
-        resolutionSelect.addItem("Mobile\t320x240");
+
+        resolutionComboBox = new JComboBox<ResolutionVideo>(
+                new ResolutionVideo[]{
+                        new ResolutionVideo("720p(HD)", 1280, 720),
+                        new ResolutionVideo("1920p", 1080, 1920),
+                        new ResolutionVideo("480p", 854, 480),
+                        new ResolutionVideo("240p", 426, 240),
+                        new ResolutionVideo("DVD", 720, 567),
+                        new ResolutionVideo("TV", 640, 480),
+                        new ResolutionVideo("Mobile", 320, 240)}
+        );
     }
 
     /**
      * Method that set all possible resolutions
      * for video converter.
      */
     protected void setFrameSelect() {
-        framesSelect.addItem("Original frame");
-        framesSelect.addItem("21F");
-        framesSelect.addItem("24F");
-        framesSelect.addItem("27F");
-        framesSelect.addItem("29,9F");
-        framesSelect.addItem("30F");
-        framesSelect.addItem("60F");
+        framesSelect=new JComboBox<FrameVideo>(
+                new FrameVideo[]{
+                        new FrameVideo(21),
+                        new FrameVideo(24),
+                        new FrameVideo(27),
+                        new FrameVideo(29),
+                        new FrameVideo(30),
+                        new FrameVideo(60)});
```

Comment:
Same here, it seems that it has to much space

Reply:
ok